.github/workflows/bench-turbopack-scheduled.yml

```
@@ -0,0 +1,138 @@
+name: Benchmark other Bundlers
+
+on:
+  schedule:
+    # Run every week on Sunday 12:42
+    - cron: "42 12 * * 0"
```

I don't think this should be on a schedule. We should be able to run this manually instead, e.g. when we've updated the version of the bundlers or added a new one.
```suggestion
  workflow_dispatch:
```

---

The bundlers itself may also change over time. e. g. a new Vite version published. I think we should ensure to track that over time.